src/components/styled/StyledLink.js

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+
+export const StyledAtag = styled.a`
```

Comment:
I think the name `StyledLink` is better to understand.

Reply:
thanks. absolute.